var/spack/repos/builtin/packages/py-crossmap/package.py

```diff
@@ -36,4 +36,5 @@ class PyCrossmap(PythonPackage):
 
     depends_on('python@2.7:2.7.999', type=('build', 'run'))
     depends_on('py-cython@0.17:', type='build')
-    depends_on('py-pysam',        type='build')
+    depends_on('py-pysam', type='build')
```

Comment:
I'm pretty sure this one should be `type=('build', 'run')`. Also, it's missing a dependency on `py-bx-python`.

Reply:
yep, fixing